Assignment8.txt

```diff
@@ -0,0 +1,9 @@
+Ashank Kumar
+
+1. printf writes text to the ouput stream(monitor) while fprintf writes text to the output stream the programmer tell it to(a file).
```

Comment:
this is all true, but you didn't include details on arguments and return values!